tests/randomize_dipnet_test.py

```diff
@@ -0,0 +1,28 @@
+from baseline_bots.randomize_dipnet import random_orders
+import daidepp
+
+class TestUtils:
+    def test(self):
+
+        # The following three tests check that build, disband and remove commands do not get changed when input into the order randomizer.
+
+        ord =  [(('FRA', "AMY", 'PAR'), 'BLD')]
+        assert ord == random_orders(ord)
+
+        ord =  [(('FRA', "AMY", 'PAR'), 'DSB')]
+        assert ord == random_orders(ord)
+
+        ord =  [(('FRA', "AMY", 'PAR'), 'REM')]
+        assert ord == random_orders(ord)
+
+        # The following three tests check that when orders that contain movements, holds, convoys and other moves get
+        # input into the order randomizer, they come out differnent
```

Comment:
I would like to see a seeded trial, where you compare the output to an actual expected string

Reply:
also different*